frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -826,13 +831,13 @@ private static function hasDuplicates(array $contents): bool {
     }
 
     /**
-     * @return array{nominations: list<array{author: array{name: null|string, username: string}, contents?: array{before_ac?: bool, difficulty?: int, quality?: int, rationale?: string, reason?: string, statements?: array<string, string>, tags?: list<string>}, nomination: string, nominator: array{name: null|string, username: string}, problem: array{alias: string, title: string}, qualitynomination_id: int, status: string, time: \OmegaUp\Timestamp, votes: list<array{time: \OmegaUp\Timestamp|null, user: array{name: null|string, username: string}, vote: int}>}|null>, pager_items: list<array{class: string, label: string, page: int}>}
-     *
      * @omegaup-request-param int $offset
      * @omegaup-request-param int $rowcount
      * @omegaup-request-param mixed $status
      * @omegaup-request-param mixed $query
      * @omegaup-request-param mixed $column
+     *
+     * @return array{nominations: list<NominationListItem|null>, pagerItems: list<PageItem>}
```

Comment:
nit:

```suggestion
     * @return array{nominations: list<NominationListItem>, pager_items: list<PageItem>}
```

todo lo que sale de los APIs debería ir en snake_case. Igual todos los demás APIs de este estilo.

para las nominaciones nulas, https://github.com/omegaup/omegaup/pull/4148 debería ayudar a que ya no sea necesario decir que pueden ser nulas.

Reply:
listo